tools/unicode_properties_parse/format_width_estimate_intervals.py

```diff
@@ -9,38 +9,30 @@
 from pathlib import Path
 
 
-LINE_REGEX = re.compile(r"([0-9A-Z]+)(\.\.[0-9A-Z]+)?;(A|F|H|N|Na|W) *#.*")
-
-
 class UnicodeWidth(Enum):
     IS_1: int = 1
     IS_2: int = 2
 
 
-class UnicodeTable:
-    # A valid Unicode code point won't exceed MAX_UNICODE_POINT.
-    MAX_UNICODE_POINT: int = 0x10FFFF
-    UNICODE_TABLE_SIZE: int = MAX_UNICODE_POINT + 1
+class UnicodeWidthTable:
+    # A valid Unicode code point won't exceed MAX_CODE_POINT.
+    MAX_CODE_POINT: int = 0x10FFFF
+    TABLE_SIZE: int = MAX_CODE_POINT + 1
 
     def __init__(self):
-        self.table = [UnicodeWidth.IS_1] * (self.UNICODE_TABLE_SIZE)
+        self.table = [UnicodeWidth.IS_1] * (self.TABLE_SIZE)
 
-    def fill_range(self, rng: tuple, width: bool):
+    # "rng" denotes a right-closed range.
+    def fill_range(self, rng: tuple, width: int):
         from_, to_ = rng
         assert from_ <= to_, "impl assertion failed"
-        assert to_ <= self.MAX_UNICODE_POINT, "impl assertion failed"
+        assert to_ <= self.MAX_CODE_POINT, "impl assertion failed"
         self.table[from_ : to_ + 1] = [width] * (to_ - from_ + 1)
 
-    def print_intervals(self):
-        """
-        Generates _Width_estimate_intervals_v2.
-        It begins from the second code point and continues up to the last one, including it as well.
-        Whenever a code point's width differs from the previous one,
-        the function print the code point to indicate the start of a new range.
-        """
+    def print_width_estimate_intervals(self):

```

Comment:
As the `[I]!=[I-1]->print I` logic and the assertion makes things very clear, with renaming I think it's enough not to add extra comments.